scripts/_common.sh

```diff
@@ -5,6 +5,10 @@
 
 pkg_dependencies="php5-gd php5-json php5-intl php5-mcrypt php5-curl php5-apcu php5-redis php5-ldap php5-imagick imagemagick acl tar smbclient"
 
+if [ "$(lsb_release --codename --short)" == "stretch" ]; then
```

Comment:
I purpose to use `"$(lsb_release --codename --short)" != "jessie"` to be compatible with the next debian version (buster). What do you think ?

Reply:
I agree.
And we will be able to use `ynh_get_debian_release` as soon as a release includes this merged PR https://github.com/YunoHost/yunohost/pull/373 :wink: